### PR TITLE
added js structure

### PR DESCRIPTION
**Pull request by caitbopp**
Added in container code for js file

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1 +1,16 @@
-//Hello teammates!
\ No newline at end of file
+$(document).ready(function () {
+
+
+
+
```
Comment (ryanascherr) [inline script.js:12]: Test comment!

File: script.js [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+
+
+
+
+
```
Comment (TLillySib) [inline script.js:22]: The document looks great!

**Review by TLillySib — CHANGES_REQUESTED** [review]
Make changes to line 4

File: script.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
 $(document).ready(function () {
 
 
-
+fetch(url)
+    .then(function(response) {
+        if (!response.ok) {
+            throw response.json();
+        }
```
Comment (TLillySib) [inline script.js:8]: Changes are accepted

File: script.js [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+
+        return response.json();
```
Comment (TLillySib) [inline script.js:10]: Test 1

**Review by TLillySib — APPROVED** [review]
Approve

**Pull request merged**